.travis.yml

```diff
@@ -1,4 +1,8 @@
-rvm: 2.1
+rvm:
+  - 2.1
+  - 2.2
```

Comment:
We should call out the real versions here not the RVM aliases that Travis doesn't update